css-builder.js

```diff
@@ -18,7 +18,11 @@ define(['require', './normalize'], function(req, normalize) {
       }
       var csslen = css.length;
       try {
-        css =  csso.justDoIt(css);
+        if (typeof csso.minify === 'function') {
+          css = csso.minify(css).css;
```

Comment:
Suggested code (`csso.minify(css).css`) will cause error in case of usage csso `v1.8.0`
```
Error: TypeError: Cannot read property 'length' of undefined
    at compress (eval at <anonymous> (/usr/local/lib/node_modules/requirejs/bin/r.js:25271:33), <anonymous>:50:63)
```
Note: tested with requirejs v `2.1.10` (as bundled in require-css) and `2.3.2` (as latest)

So we need to improve code a bit, so that to support at least 3 major versions.